### ISSUE TITLE
Migrate d3 treemaps to grafana treemaps

### ISSUE THREAD

**Issue by csaladenes** [denesdata/roeim#37]
https://grafana.com/grafana/plugins/marcusolsson-treemap-panel

**Comment by csaladenes** [comment]
Depends on #51

**Comment by csaladenes** [comment]
`wontfix` as the `Grafana` treemap is yet far inferior to the `d3plus` one.
